ml/backend.go

```diff
@@ -119,6 +119,25 @@ type Context interface {
 	Layer(int) Context
 }
 
+// RopeOpts contains optional parameters for RoPE function
+type RopeOpts struct {
```

Comment:
I'd prefer no abbreviations here
```suggestion
type RopeOptions struct {
```